### ISSUE TITLE
seems like the uploaded file is not passed to the notebook?

### ISSUE THREAD

**Issue by volodymyrss** [oda-hub/dispatcher-app#687]
the default parameter is passed instead. Could you please have a look?

![image](https://github.com/oda-hub/dispatcher-app/assets/3909535/55a7d043-cab1-480d-a695-fdc4d5229426)

**Comment by volodymyrss** [comment]
Actually seems like another parameter is also not passed. So I add @dsavchenko

**Comment by dsavchenko** [comment]
The problem with a second parameter might be caused by the fact that it's declared as int, but is annotated as float. https://github.com/oda-hub/nb2workflow/pull/155 will address this situation. I added there another TOOD point to have a dedicated test.
This guess is to verify, of course.

**Comment by volodymyrss** [comment]
> The problem with a second parameter might be caused by the fact that it's declared as int, but is annotated as float. [oda-hub/nb2workflow#155](https://github.com/oda-hub/nb2workflow/pull/155) will address this situation. I added there another TOOD point to have a dedicated test. This guess is to verify, of course.

Yes, I noticed this too and changed to float, so it works. Annoying that it's silently dropped, good that there is a fix.
The problem with the file remains.

**Comment by dsavchenko** [comment]
> Annoying that it's silently dropped, good that there is a fix.

There is no fix for "silently drop" behaviour yet. It only happens in async, need to trace it down. I will try to do it in the "multiline" PR as well

**Comment by burnout87** [comment]
There is a small bug on the tab-generator

![image](https://github.com/oda-hub/dispatcher-app/assets/3757833/ae2f49c7-54a9-4133-8694-f0dcd269b7f8)

**Comment by burnout87** [comment]
https://github.com/oda-hub/frontend-tab-generator/pull/39

**Comment by burnout87** [comment]
ok after updating the dispatcher-container (https://github.com/oda-hub/dispatcher-container/pull/12), the file is:

1. uploaded by the dispatcher
2. download url properly generated by the dispatcher
3. file downloaded by the nb2workflow service
4. file used inside the notebook

![image](https://github.com/oda-hub/dispatcher-app/assets/3757833/9146643f-0bb3-4b5d-9740-9ebad3832f14)

**Comment by volodymyrss** [comment]
increase allowed size to 500Mb (in nb2w)
and if the exception is happening but the product is ready. it should not proceed with execution if the exception happenned
